webapp/graphite/render/views.py

```diff
@@ -78,6 +78,12 @@ def renderView(request):
 
   # Now we prepare the requested data
   if requestOptions['graphType'] == 'pie':
+    targets = [target for target in requestOptions['targets'] if target.find(':') < 0]
```

Comment:
Looking good, let's just move this inside the `if` to avoid doing the extra work if we don't need it.